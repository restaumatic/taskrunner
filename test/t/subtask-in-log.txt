# no toplevel

TASKRUNNER_DEBUG=1 taskrunner -n foo bash -c '
  taskrunner -n bar echo hello

  # again, skipped
  taskrunner -n bar echo hello
' 2>&1 | grep -E '([|] (Subtask|Starting subtask|hello))'
