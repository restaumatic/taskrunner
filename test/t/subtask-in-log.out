-- output:
[foo] debug | Starting subtask bar
[bar] stdout | hello
[foo] debug | Subtask bar finished with ExitSuccess
[foo] debug | Starting subtask bar
[foo] debug | Subtask bar finished with ExitSuccess
